blogposts/2022/release-3.36.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'Sourcegraph 3.36 release'
+publishDate: 2022-01-21T10:00-07:00
+description: 'Sourcegraph 3.36 introduces Notebooks and query-based search contexts in Beta, a new file fuzzy finder, and the ability to push branches from batch changes to forks.'
+tags: [blog, release]
+slug: 'release/3.36'
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.36/sourcegraph-3-36-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.36/sourcegraph-3-36-release.png
+changelogItems:
+  - description: 'Search contexts can now be defined with a restricted search query, allowing them to be used for subsections of monorepos or selective groupings of repos. Allowed filters: `repo`, `rev`, `file`, `lang`, `case`, `fork`, `visibility`. `OR`, `AND` expressions are also allowed. This is a `Beta` feature and may change in following releases.'
+    url: https://docs.sourcegraph.com/code_search/how-to/search_contexts#beta-query-based-search-contexts
+    category: Search
+  - description: 'Batch changes will be created on forks of the upstream repository if the new `batchChanges.enforceForks` site setting is enabled, allowing users with restricted repository access to take advantage of Batch Changes.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/17879
+    category: Batch Changes
+  - description: 'Notebooks are now fully persistable and can be shared via a unique URL. We support two visibility modes: private (only the creator can view the notebook) and public (everyone can view the notebook). This is a `Beta` feature and may change in following releases.'
+    url: https://github.com/sourcegraph/sourcegraph/issues/27384
+    category: Search
+  - description: 'Search results across repositories are now ordered by repository rank to make results more relevant. By default the rank is the number of stars a repository has. An administrator can inflate the rank of a repository via `experimentalFeatures.ranking.repoScores`. If you notice increased latency in results, you can disable this feature by setting `experimentalFeatures.ranking.maxReorderQueueSize` to 0.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/29856
+    category: Search
+  - description: 'Symbolic links are now searchable to make files easier to find. Previously it was possible to navigate to symbolic links in the repository tree view, however the symbolic links were ignored during searches.'
+    url: https://github.com/sourcegraph/zoekt/pull/237
+    category: Search
+  - description: "Sourcegraph's API (streaming search, GraphQL, etc.) may now be used from any domain when using an access token for authentication, or with no authentication in the case of Sourcegraph.com."
```

Comment:
I initially thought this opened our API to public usage, but it was already public with restrictions, and this seems to be more of a security change. Checking in with the engineering team, but it might make sense to remove if there's no impact to customers.

Reply:
Can't find any use cases or direct impact to existing customers for this one, so I'm removing for now.